.circleci/config.yml

```diff
@@ -40,6 +40,7 @@ run_e2e_and_save_artifacts: &run_e2e_and_save_artifacts
         environment:
           JUNIT_REPORT_PATH: ./junit/
           FRONTEND_TEST_USE_LOCAL_DIST: true
+          NO_PARALLEL: true
```

Comment:
this is really weird but it's the only way i could get circleCI to build.  something to do with webpack4's new minification plugin.  just doesn't work by default on CircleCI.  when i shut off parrelization, it works.

Reply:
@alisman odd that it only is required for `serveDist` and `yarn run test`. Or maybe it should be set for the run build command as well, but wasn't caught b/c of CircleCI caching?